localdev/docker-compose.yaml

```diff
@@ -4,6 +4,9 @@ networks:
   kind:
     external: true
 
```

Comment:
Can we remove these changes from this PR?

Reply:
oops, yes will remove -- should be in master now